stestr/commands/history.py

```diff
@@ -0,0 +1,389 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+"""Interact with the run history in a repository."""
+
+import functools
+import sys
+
+from cliff import command
+import prettytable
```

Comment:
How about using cliff lister instead of using prettytable directly? That has formatters like csv, value, etc.

```suggestion
from cliff import lister
```

Reply:
That's a good call, we definitely should just use the cliff lister. This was left over from the first commit on this PR where I was manually building sub-commands via argparse, and using a lister wasn't applicable there because there was only a single command. But since moving to separate cliff commands there's no reason to do this manually anymore.